### ISSUE TITLE
Medusa doesn't start with release 0.2.7

### ISSUE THREAD

**Issue by Rwanou88** [pymedusa/Medusa#4767]
**Describe the bug**
Medusa doesn't start

**To Reproduce**
Update Medusa with the last release 0.2.7.

**Expected behavior**
Since I have updated Medusa, I have a black screen when I want to access it.

**Screenshots**
![error](https://user-images.githubusercontent.com/30654264/43314031-0244964c-9192-11e8-9b01-20dee9598c87.png)
![ressources](https://user-images.githubusercontent.com/30654264/43314032-025931ce-9192-11e8-8b66-e4d9c3815ba4.png)
![warning](https://user-images.githubusercontent.com/30654264/43314033-026eba08-9192-11e8-8dfc-e03f2f64f31f.png)

**Medusa (please complete the following information):**
 - OS: DSM 6.2-23739 Update 2 
 - Branch: master
 - Commit: b7333559c9a747cd3e8b4022c4073757a62b32ed

**Logs:**
<details>
```
```
</details>

**Additional context**
Nop

**Comment by kantjer** [comment]
I found that when starting Medusa in Chrome's Incognito mode it works.
For now I use that  as a workaround.

**Comment by p0psicles** [comment]
@OmgImAlexis this seems to be an issue with the browser local storage. We really need to fix this. Can you look into this?

**Comment by medariox** [comment]
Clearing the cache should fix this, see: https://github.com/pymedusa/Medusa/issues/4754#issuecomment-407949046

**Comment by Rwanou88** [comment]
It's seems to be a cache problem indeed, working on incognito mode ;-)

**Comment by p0psicles** [comment]
Yeah, but when restarting medusa, it should provide you new js files.
Also the mako cache should have been refreshed. I asume you tried restarting medusa, and closed all tabs (as in not reusing old tabs). Then the only item I can think of that will survive that, is the local storage. Local storage is not shared with incognito tabs.

**Comment by OmgImAlexis** [comment]
@p0psicles isn't much we can do. All users will need to clear all caches and we can't really do that programatically.

**Comment by p0psicles** [comment]
Can you go through it with @Rwanou88 so we are 100% sure howto resolve this issue? Then we at least know for sure.

**Comment by OmgImAlexis** [comment]
> Working fine now I cleared the cache.

Check the comment, they updated it.
